api/document/js/__tests__/example-doc.test.ts

```diff
@@ -0,0 +1,5 @@
+import exampleDoc from "../example-doc";
+
+test('example document conforms to schema', () => {
+  exampleDoc.check();
```

Comment:
This is admittedly a silly test, since we'll be getting rid of the example document soon, but we can always modify this later on to contain example documents or somesuch (and probably rename the suite to `schema.test.js`, as a way to sanity check the schema itself).